src/pages/AdvanceQuarry.jsx

```diff
@@ -0,0 +1,25 @@
+import { Flex, Text } from "@sparrowengg/twigs-react";
+
+const AdvanceQuarry = () => {
+  return (
+    <Flex
+      css={{
+        width: "1080px",
```

Comment:
@9xVibee is the fixed width is Needed?????

Reply:
Its just added because advance quaary and Events dont have the content and after applying routing when we will navigate the UI should not get change that's why added the fix width!